crates/cdk-cli/src/sub_commands/dlc/mod.rs

```diff
@@ -74,7 +74,7 @@ pub struct UserBet {
     dlc_root: String,
     timeout: u64,
     amount: u64,
-    locked_ecash: Option<String>, // needs to be a new struct with x and Z'
+    locked_ecash: Option<Vec<TokenV3>>, 
```

Comment:
I was thinking we should use a new struct called `DLCFundingToken` that is the same thing but has the extra `dlc_root` field. I guess its not really necessary at this point because we include the root in the the `UserBet`

Reply:
Right... that was my thinking.  Since we don't need to send the mint everything in `UserBet`,  `DLCFundingToken` may work for the message we send to the mint which needs to include the locked notes, the root hash and the total funding amount.